tools/data/activitynet/convert_proposal_format.py

```diff
@@ -0,0 +1,160 @@
+"""This file converts the output proposal file of proposal generator (BMN) into
+the input proposal file of action classifier (SSN)."""
```

Comment:
Is this script restricted to BMN and SSN?

Reply:
No, they are just examples.

Reply:
If possible, provide a full list of supported input/output methods.

Reply:
OK.